### PR TITLE
Update packages 03 20

### PR DESCRIPTION
**Pull request by nikolaevtomov**
Trello: https://trello.com/c/v23SDBT7/44-05-redux-requests-fix-peer-dependencies

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -34,24 +34,24 @@
...
+    "jest": "^24.9.0",
+    "prettier": "^1.19.1",
+    "request": "^2.88.2",
+    "ts-jest": "^24.3.0",
+    "tslint": "^5.20.1",
+    "tslint-config-dabapps": "^0.6.2",
+    "typescript": "^3.8.3"
   },
   "peerDependencies": {
     "redux-thunk": "*",
```
Comment (isidornygren) [inline package.json]: Since you're not changing the peer dependency versioning, is bumping the redux version fixing this issue?

Comment (isidornygren) [reply package.json]: (As in the issue described in the trello card)

File: package.json [hunk 2]
```diff
@@ -34,24 +34,24 @@
   },
   "homepage": "https://github.com/dabapps/redux-requests#readme",
   "dependencies": {
-    "@types/js-cookie": "^2.0.28",
-    "@types/node": "^7.0.18",
-    "axios": "^0.18.0",
+    "@types/js-cookie": "^2.2.5",
+    "@types/node": "^10.17.17",
```
[the remark this answers is not in the document]
Comment (nikolaevtomov) [reply package.json]: good spot

File: package.json (hunk 1 above)
Comment (nikolaevtomov) [reply package.json]: That is right, it fixes the issue.

File: package.json [hunk 3]
```diff
@@ -34,24 +34,24 @@
...
+    "axios": "^0.18.1",
     "flux-standard-action": "^1.2.0",
-    "js-cookie": "^2.1.4",
+    "js-cookie": "^2.2.1",
     "path": "^0.12.7",
-    "redux": "*",
+    "redux": "^4.0.5",
     "redux-thunk": "*"
```
[the remark this answers is not in the document]
Comment (nikolaevtomov) [reply package.json]: I don't think we should restrict the version here, as some project may still use the older version.
Any objection?

[the remark this answers is not in the document]
Comment (nikolaevtomov) [reply package.json]: Oh wait, this is dependent, yes I should change it!

[the remark this answers is not in the document]
Comment (JakeSidSmith) [reply package.json]: This should remain `*`.

File: package.json [hunk 4]
```diff
@@ -34,24 +34,24 @@
...
-    "tslint-config-dabapps": "github:dabapps/tslint-config-dabapps#v0.5.3",
-    "typescript": "^3.3.4000"
+    "@types/jest": "^24.9.1",
+    "jest": "^24.9.0",
+    "prettier": "^1.19.1",
+    "request": "^2.88.2",
+    "ts-jest": "^24.3.0",
+    "tslint": "^5.20.1",
+    "tslint-config-dabapps": "^0.6.2",
+    "typescript": "^3.8.3"
```
Comment (JakeSidSmith) [inline package.json]: Please put this back to `3.3.4000`. `3.4+` is not backward compatible.

File: package.json [hunk 5]
```diff
@@ -34,24 +34,24 @@
...
+    "@types/js-cookie": "^2.2.5",
+    "@types/node": "^12.12.31",
+    "axios": "^0.18.1",
     "flux-standard-action": "^1.2.0",
-    "js-cookie": "^2.1.4",
+    "js-cookie": "^2.2.1",
     "path": "^0.12.7",
-    "redux": "*",
-    "redux-thunk": "*"
+    "redux": "^4.0.5",
```
Comment (JakeSidSmith) [inline package.json]: Should be `*`

File: package.json [hunk 6]
```diff
@@ -34,24 +34,24 @@
...
+    "redux-thunk": "^2.3.0"
```
Comment (JakeSidSmith) [inline package.json]: Should be `*`

**Pull request merged**